src/lib/unregistered-services-healthCheck.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 		return {
 			getStatus: () => {
 				return {
-					name: `All services for ${appName} are registered in Next-metrics`,
+					name: `Metrics: All services for ${appName} registered in next-metrics`,
```

Comment:
Just a small thing, should it not be: Metrics: All services for ${appName} **are** registered in next-metrics?

Reply:
No, because when it fails that `are` makes it look like it's a problem that they are registered.

Reply:
Oh right, that was the thing you mentioned yesterday, sorry.